packages/js/src/ui/components/elements/Markdown.tsx

```diff
@@ -0,0 +1,71 @@
+import { createMemo, For, JSX } from 'solid-js';
+import { useStyle } from '../../helpers';
+
+interface Token {
+  type: 'bold' | 'text';
+  content: string;
+}
+
+const parseMarkdownIntoTokens = (text: string): Token[] => {
+  const tokens: Token[] = [];
+  let buffer = '';
+  let inBold = false;
+
+  for (let i = 0; i < text.length; i += 1) {
+    // Check if it's an escaped character
+    if (text[i] === '\\' && text[i + 1] === '*') {
+      buffer += '*';
+      i += 1;
+    }
+    // Check for bold marker **
+    else if (text[i] === '*' && text[i + 1] === '*') {
+      if (buffer) {
+        tokens.push({ type: inBold ? 'bold' : 'text', content: buffer });
+        buffer = '';
+      }
+      inBold = !inBold;
+      i += 1;
+    } else {
+      buffer += text[i];
+    }
+  }
+
+  // Push any remaining buffered text as a token
+  if (buffer) {
+    tokens.push({ type: inBold ? 'bold' : 'text', content: buffer });
+  }
+
+  return tokens;
+};
+
+const Bold = (props: { children?: JSX.Element }) => {
+  const style = useStyle();
+
+  return <span class={style('markdownBold', 'nt-font-semibold')}>{props.children}</span>;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** Should we use `<strong>` instead of `<span>`? As `<strong>` seems to be a standard in markdown formatters‏

Reply:
strong just didn't look good. you mean do <strong> and apply another font weight ?